Assets/MixedRealityToolkit/Inspectors/TextureCombinerWindow.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.MixedReality.Toolkit.Editor
 {
-    public class ChannelPackerWindow : EditorWindow
+    public class TextureCombinerWindow : EditorWindow
```

Comment:
Does this class name change have anticipated customer impact?

Reply:
Good question, it does not unless someone changed the private methods to public and incorporated into some pipeline. But, I don't think many developers know this tool exists yet.

Reply:
feels safe.